pkg/daemon/ceph/osd/volume.go

```diff
@@ -89,6 +89,26 @@ type cephVolVg struct {
 	Devices string `json:"devices"`
 }
 
+type cephVolReportV2 struct {
+	BlockDB      string `json:"block_db"`
+	Encryption   string `json:"encryption"`
+	Data         string `json:"data"`
+	DatabaseSize string `json:"data_size"`
+	BlockDbSize  string `json:"block_db_size"`
+}
+
+func isNewStyledLvmBatch(version cephver.CephVersion) bool {
+	if version.IsNautilus() && version.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 13}) {
+		return true
+	}
+
+	if version.IsOctopus() && version.IsAtLeast(cephver.CephVersion{Major: 15, Minor: 2, Extra: 8}) {
+		return true
+	}
+
+	return false
```

Comment:
If Pacific or newer we should return true, right?

Reply:
seems 16.0.0 still uses legacy format ...

Reply:
That’s unexpected, the changes should be in master first, oh well we can revisit separately.